hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modal {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.modalGrid {
+    background-color: white;
+    padding: 1em;
+    @include responsive(sm-down) {
```

Comment:
for mobile screens, set screen size to a variable width, 85% looked good to me while I was testing. Also move your @include statement to the bottom of the class, it just looks better that way.

Reply:
done